crates/vise/Cargo.toml

```diff
@@ -12,7 +12,7 @@ keywords.workspace = true
 categories.workspace = true
 
 [dependencies]
-vise-macros = { version = "0.1.0", path = "../vise-macros" }
+vise-macros = { version = "=0.1.1", path = "../vise-macros" }
```

Comment:
I think 0.1.0 will get broken because it didn't pin the `vise-macros` version. Not sure what we can do about it.

Reply:
Maybe release as 0.2.0 then?